FreeSurfer/FreeSurferPipeline.sh

```diff
@@ -369,6 +395,9 @@ get_options()
 	if [ ! -z "${p_conf2hires}" ]; then
 		log_Msg "Include -conf2hires flag in recon-all: ${p_conf2hires}"
 	fi
+	if [ "${p_t2w_image}" = "NONE" ] ; then
+  		Compliance="legacy"
```

Comment:
This appears to set the wrong value, and seems like it will never get used as it is after the checking call.  However, this variable appears to already be checked above.

Reply:
However, for consistency with the other "optional parameters", we should check whether `p_mppmode` is defined (which it will be, because we set its default above) and then message its value with a `log_Msg` statement.

Reply:
I left the check for p_mppmode so that it will ony be reported when actually set.